packages/utils/test/unit/format.test.ts

```diff
@@ -0,0 +1,19 @@
+import {describe, it, expect} from "vitest";
+import {toChecksumAddress} from "../../src/index.js";
+
+describe("toChecksumAddress", () => {
```

Comment:
should at least add all tests cases from the ERC, and maybe a few additional ones with invalid input

```
# All caps
0x52908400098527886E0F7030069857D2E4169EE7
0x8617E340B3D01FA5F11F306F4090FD50E238070D
# All Lower
0xde709f2102306220921060314715629080e2fb77
0x27b1fdb04752bbc536007a920d24acb045561c26
# Normal
0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed
0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359
0xdbF03B407c01E7cD3CBea99509d93f8DDDC8C6FB
0xD1220A0cf47c7B9Be7A2E6BA89F429762e7b9aDb
```

Could write the tests in a TestCase[] pattern to reduce boilerplate

https://github.com/ChainSafe/lodestar/blob/fa30bcf10086292883ef086202474ac4df67ffe1/packages/utils/test/unit/retry.test.ts#L17